.github/workflows/publish-bicep.yaml

```diff
@@ -21,88 +21,165 @@ on:
       - main
     tags:
       - v*
-  pull_request:
-    branches:
-      - main
-  workflow_dispatch:
+  workflow_run:
+    workflows: ["Approve Publish Bicep Types"]
+    types:
+      - completed
+    workflow_dispatch:
     inputs: {}
 
 permissions:
       id-token: write
       contents: read
+      checks: write
 
 env: 
   # bicep-types ACR url for uploading AWS Bicep types
   BICEP_TYPES_REGISTRY: 'biceptypes.azurecr.io'
   AWS_REGION: us-west-2
   CI_PUBLISH_RELEASE: ${{ github.repository == 'radius-project/bicep-types-aws' && startsWith(github.ref, 'refs/tags/v') && github.event_name == 'push' }}
   CI_PUBLISH_LATEST: ${{ github.repository == 'radius-project/bicep-types-aws' && github.ref == 'refs/heads/main' && github.event_name == 'push' }}
+  PUBLISH_BICEP_APP_ID: 1077084
```

Comment:
Do we need to document this to build shared context?

Reply:
Yes, we should. Will open a task to track